drivers/SmartThings/philips-hue/src/handlers.lua

```diff
@@ -60,6 +60,22 @@ local function do_switch_level_action(driver, device, args)
     return
   end
 
+  local is_off = device:get_latest_state(
+    "main", capabilities.switch.ID, capabilities.switch.switch.NAME) == "off"
+
+  if is_off then
+    local resp, err = hue_api:set_light_on_state(light_id, true)
+    if not resp or (resp.errors and #resp.errors == 0) then
+      if err ~= nil then
+        log.error("Error performing switch level action: " .. err)
```

Comment:
should this error message indicate the capability command that had the error (as you've done here) or the api command that had the error? Fine with either answer.

Reply:
No that's a copy paste oopsie, I'll address it.